cli/command/trust/signer_add_test.go

```diff
@@ -99,7 +99,9 @@ func TestSignerAddCommandInvalidRepoName(t *testing.T) {
 
 	assert.NoError(t, cmd.Execute())
 
-	expectedError := fmt.Sprintf("Failed to add signer to %s: invalid repository name (%s), cannot specify 64-byte hexadecimal strings\n", imageName, imageName)
+	expectedError := fmt.Sprintf("\nAdding signer \"alice\" to %s...\n",
```

Comment:
should the command be erroring on the line above?

Reply:
I don't think so -- because we want to keep going to the next repo specified (e.g. multi-repo add), we only print the failure message instead of erroring out:

```			fmt.Fprintf(cli.Out(), "Failed to add signer to %s: %s\n", imageName, err)
```